packages/scss/src/components/form/index.scss

```diff
@@ -3,6 +3,7 @@
 @use 'exports' as *;
 
 .form {
```

Comment:
You can slightly optimize the whole thing with :
```
.form,
.form-group {
   @include vars;
}
```

Reply:
True, but it would lead to have multiple form-group declarations.
Let's improve the whole thing during the next refactor :)